core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -487,17 +489,37 @@ public List<Address> getWorkerAddresses() {
     return mWorkerConfigStore.getLiveNodeAddresses();
   }
 
+  @Override
+  public List<MasterInfo> getMasterInfoList() {
+    List<MasterInfo> masterInfoList = new ArrayList<>(mMasters.size());
+    for (MetaMasterInfo master : mMasters) {
+      masterInfoList.add(new MasterInfo(master.getId(),
+          master.getAddress(), master.getLastUpdatedTimeMs()));
+    }
+    return masterInfoList;
+  }
+
+  @Override
+  public List<MasterInfo> getLostMasterInfoList() {
+    List<MasterInfo> masterInfoList = new ArrayList<>(mLostMasters.size());
+    for (MetaMasterInfo master : mLostMasters) {
+      masterInfoList.add(new MasterInfo(master.getId(),
+          master.getAddress(), master.getLastUpdatedTimeMs()));
+    }
+    return masterInfoList;
+  }
```

Comment:
i don't really understand why the array->List conversion is needed

Reply:
According to MasterInfo class is changed into MetaMasterInfo, there maybe some differences on params between Wire.MasterInfo and Meta.MetaMasterInfo. So this function is needed.

Reply:
I don't really think there's change in params. Can we change this to returning a copied `MasterInfo[]`?

Reply:
Actually we may add package information for master, which metaMasterInfo may not use it.

Reply:
Fixed